scripts/show_test_coverage_for_files_changed_in_last_month.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+#####
+# Generates a rough report of files changed in the last month, and their test coverage
+#
+# This report could help people who like writing tests find recently changed regions of code that need tests.
+#
+# usage: From the root directory of the repository, run this script
+# e.g: $ ./scripts/show_test_coverage_for_files_changed_in_last_month.sh
+#####
+
+#make cover directory if it does not exist, so we can place output in a folder excluded by .gitignore
+mkdir -p ./cover/
+
+git diff HEAD '@{last month}' --name-only > ./cover/files_changed_in_last_month.txt
+
+# append an additional search string that aligns with the coverage header
+# so the header is also emitted and highlighted
+echo "LINES RELEVANT   MISSED" >> ./cover/files_changed_in_last_month.txt
+
+mix test --cover > ./cover/test_coverage_by_file.txt
```

Comment:
I would add the flag `--exclude needs_attention`, as it's done in the CI.
A bunch of tests have been marked as such. There are quite a few, and many of them are far from trivial to fix. Many of them are also about the spring protocol and I don't want to spend too much time on them since we want to deprecate the protocol. So in order to get a useful CI, they are now skipped.

Reply:
Added `--exclude needs_attention`